src/datafactory/azext_datafactory/generated/_help.py

```diff
@@ -19,28 +19,28 @@
 
 helps['datafactory factory list'] = """
     type: command
-    short-summary: Lists factories under the specified subscription.
+    short-summary: "Lists factories under the specified subscription."
```

Comment:
Is the double quotes added to solve some issue?

Reply:
Yes. we have found issues of parsing the command help info when the description has : in it.